### DISCUSSION TITLE
🚀 card-index.com - NiceGUI live in production

### DISCUSSION THREAD

**Discussion by adarshpunj** [zauberzeug/nicegui#2717] [Show and tell]
A few months back I got to know about NiceGUI and almost immediately started using it, and even made a contribution as well in [1.4.2](https://github.com/zauberzeug/nicegui/releases/tag/v1.4.2)

Yesterday I pushed a full fledged app at [card-index.com](https://card-index.com) (kinda IMDb but for credit cards) built in NiceGUI.

Stack:
- NiceGUI
- Postgres
- GPT (automation)

**Comment by rodja** [comment]
Very nice. Would you like to add it to our list of comuity projects in https://github.com/zauberzeug/nicegui/wiki#community-projects?

**Reply by adarshpunj** [reply]
@rodja sure, I'll add. Thx!
